schemas/multimodal/build.py

```
@@ -178,34 +180,70 @@ def build_contracts(
+        ]
+        missing_outputs = [
+            output_path
+            for output_path in expected_outputs
+            if not output_path.exists()
+        ]
+        if missing_outputs:
+            missing = ", ".join(str(path) for path in missing_outputs)
             raise FileNotFoundError(
-                f"Missing generated Python protobuf module: {python_generated}"
+                f"Missing generated protobuf output(s): {missing}"
             )
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)